### PR TITLE
feature/APPEALS-50882 - Open Telemetry configuration and implementation

### PR DESCRIPTION
**Pull request by raymond-hughes**
Resolves [Open Telemetry configuration and implementation
](https://jira.devops.va.gov/browse/APPEALS-50882)

# Description
Implements Open Telemetry to allow Dynatrace usage of RUM, Tracing, and Data Explorer.

## Acceptance Criteria
- [x] Code compiles correctly

## Testing Plan

1. Go to [APPEALS-55730](https://jira.devops.va.gov/browse/APPEALS-55730)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -122,7 +140,7 @@ group :test, :development, :demo do
   gem "rubocop-performance"
   gem "rubocop-rails"
   gem "scss_lint", require: false
-  gem "simplecov", git: "https://github.com/colszowka/simplecov.git", require: false
+  # gem "simplecov", git: "https://github.com/colszowka/simplecov.git", require: false
```
Comment (mikefinneran) [inline Gemfile:143]: No more simplecov?

Comment (raymond-hughes) [reply Gemfile:143]: It is giving us an error when we bundle for some reason. We are going to figure out whats going on with it

Comment (raymond-hughes) [reply Gemfile:143]: This is being re-activated

File: config/initializers/open_telemetry.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+if !Rails.env.development? && !Rails.env.test? && !Rails.env.demo?
+  OpenTelemetry::SDK.configure do |c|
+    c.service_name = 'ruby-quickstart'
+    c.service_version = '1.0.1'
+
+    c.use 'OpenTelemetry::Instrumentation::ActiveRecord'
+    c.use 'OpenTelemetry::Instrumentation::Rack', { untraced_endpoints: ['/health-check', '/sample', '/logs'] }
+    c.use 'OpenTelemetry::Instrumentation::Rails'
+
+    # c.use 'OpenTelemetry::Instrumentation::PG'
```
Comment (ronwabVa) [inline config/initializers/open_telemetry.rb:40]: clean up?

**Review by mikefinneran — APPROVED** [review]

**Pull request merged**